src/script/api/script_cargolist.cpp

```diff
@@ -31,7 +31,7 @@ ScriptCargoList_IndustryAccepting::ScriptCargoList_IndustryAccepting(IndustryID
 	Industry *ind = ::Industry::Get(industry_id);
 	for (uint i = 0; i < lengthof(ind->accepts_cargo); i++) {
 		CargoID cargo_id = ind->accepts_cargo[i];
-		if (cargo_id != CT_INVALID) {
+		if (IsValidCargoID(cargo_id)) {
```

Comment:
```suggestion
		if (::IsValidCargoID(cargo_id)) {
```
IIRC we use `::` when calling non script functions